docs/zh/latest/plugins/kafka-logger.md

```diff
@@ -62,7 +62,44 @@ title: kafka-logger
 - **default**:
 
     ```json
-    {"upstream":"127.0.0.1:1980","start_time":1602211788041,"client_ip":"127.0.0.1","service_id":"","route_id":"1","request":{"querystring":{"ab":"cd"},"size":90,"uri":"\/hello?ab=cd","url":"http:\/\/localhost:1984\/hello?ab=cd","headers":{"host":"localhost","content-length":"6","connection":"close"},"body":"abcdef","method":"GET"},"response":{"headers":{"content-type":"text\/plain","server":"APISIX\/1.5","connection":"close","transfer-encoding":"chunked"},"status":200,"size":153},"latency":99.000215530396}
+    {
```

Comment:
Please also do the same change to docs/eh/latest/plugins/kafka-logger.md

Reply:
Thanks for the reminder, the documentation has been added